downstream/action.yml

```
@@ -61,9 +64,13 @@ runs:
       if: ${{ ! inputs.token }}
       id: sts
       with:
-        scope: ${{ inputs.repo }}
+        scope: ${{ inputs.token-scope || inputs.repo }}
         identity: ${{ inputs.identity }}
 
+    - shell: bash
+      run: |
+        git config "http.https://github.com/".extraheader "AUTHORIZATION: basic $(echo -n "x-access-token:${{ steps.sts.outputs.token }}" | base64)"
```

```suggestion
        git config "http.https://github.com/".extraheader "AUTHORIZATION: basic $(echo -n "x-access-token:${{ steps.sts.outputs.token }}" | base64 -w0)"
```

Otherwise we might get multiple lines of output, could cause problems down the line